src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/mgmt_iot_hub_device/lib/iot_hub_device_client.py

```diff
@@ -58,7 +58,9 @@ def __init__(
         if accept_language is not None and not isinstance(accept_language, str):
             raise TypeError("Optional parameter 'accept_language' must be str.")
         if not base_url:
-            base_url = 'https://example.azure-devices.net'
+            from azure.cli.core._profile import CLOUD
```

Comment:
Maybe I missed it but does this still exist?
I think it was removed as I don't see it here - https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/_profile.py

Reply:
You're right, it is removed. You would do `cli_ctx.cloud` to get the reference. Good catch.